eos/b-decays/b-to-d-l-nu.cc

```diff
@@ -65,7 +69,7 @@ namespace eos
                 // only B_{d,u} mesons can decay in this channel
                 throw InternalError("BToDLeptonNeutrino: q = '" + o["q"] + "' is not a valid option for this decay channel");
             }
-
+            
```

Comment:
You seem to have introduced some form of whitespace (tabulator perhaps?). Can you try to remove this?